docs/index.rst

```diff
@@ -1,19 +1,85 @@
-Welcome to mpl-interactions's documentation!
-===============================================
+Welcome to mpl-interactions' documentation!
+===========================================
+
+
+mpl-interactions' design creates helpful ways of interacting with `Matplotlib <http://www.matplotlib.org>`_ plots. 
+The library provides three key types of utility for easy-to-use, accessible infographics. These modules 
+include Generic, Utils, and Jupyter.
```

Comment:
heads up I don't think that infographic is quite the right term. infographic feel static and also like something you'd present to someone else. In contrast this library provides interactions and is much more useful for the exploratory phase of a project when you are trying to understand how a function changes as a result of a paramter or visualize your data interactively. 

I've struggled always to find a brief way to actually explain what it is for. Maybe:

`mpl_interactions` removes the boilerplate from making interactive Matplotlib plots. With it you can easily:
- change the parameters of plotted functions using `ipywidgets` sliders
- compare horizontal/vertical slices of heatmaps with the mouse
- scroll to zoom for 2D plots